packages/shared/src/hooks/post/useSmartTitle.ts

```diff
@@ -103,7 +103,7 @@ export const useSmartTitle = (post: Post): UseSmartTitle => {
   const fetchSmartTitle = useCallback(async () => {
     if (!fetchedSmartTitle) {
       const smartTitlePost = {
-        ...post,
+        ...(post.sharedPost ? post.sharedPost : post),
```

Comment:
Shouldn't this include the logic if commentary is empty or not?

Reply:
`fetchTranslations` should ignore it if no title, so then below `setQueryData` won't execute

Reply:
this was a fix for that issue you found where it would always try to update top level post cache, instead of shared one if it exists

Reply:
Oh I see :) 
It seems to work good either way.